src/Helper/Pagination.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Moip\Helper;
+
+class Pagination
```

Comment:
This file needs to be documented, even though the function names are fairly self explanatory :)